daemon/policy.go

```diff
@@ -190,42 +189,6 @@ type AddOptions struct {
 	Replace bool
 }
 
-func (d *Daemon) policyAdd(rules policyAPI.Rules, opts *AddOptions, prefixes []*net.IPNet) (uint64, error) {
-	d.policy.Mutex.Lock()
-	defer d.policy.Mutex.Unlock()
-
-	oldRules := policyAPI.Rules{}
-
-	if opts != nil && opts.Replace {
```

Comment:
Are you sure that we can just remove the replace logic?

Reply:
is in there, but in the `PolicyAdd` function.